builtin/common/misc_helpers.lua

```diff
@@ -168,7 +168,7 @@ function string.split(str, delim, include_empty, max_splits, sep_is_pattern)
 	delim = delim or ","
 	max_splits = max_splits or -1
 	local items = {}
-	local pos, len, seplen = 1, #str, #delim
+	local pos, len, #str, #delim
```

Comment:
Should be:
`local pos, len = 1, #str`

Reply:
No, if you want to remove `seplen` then it should be `local pos, len = 1, #str` because the values after the equal sign have the same order as the defined variables.

Reply:
Done.